app/src/pages/playground/MessageRolePicker.tsx

```diff
@@ -3,6 +3,7 @@ import { css } from "@emotion/react";
 
 import { Item, Picker } from "@arizeai/components";
 
+import { isChatMessageRole } from "@phoenix/pages/playground/playgroundUtils";
```

Comment:
since it's in the same folder it might be just easier to do "./playgroundUtils";

Reply:
Yeah agree. Our vs code workspace settings are defaulting to route alias for auto import however.

There is a workspace setting that prefers sibling import and then aliases. I'll play with that

Reply:
Fixed